.circleci/config.yml

```
@@ -2,7 +2,7 @@ version: 2
 jobs:
   validate:
     docker:
-      - image: trussworks/circleci-docker-primary:tf12-cb0bcbc3cb9b8b2465ec5721386771769ecaa9bb
+      - image: trussworks/circleci-docker-primary
```

I think we still want to keep this pinned to a hash 

```suggestion
      - image: trussworks/circleci-docker-primary:c3cdcb24c00afc1ba635f8f6efd215f55aaf2079
```

---

Do we? I like the idea of using latest. I mean, how often does this container really change?